modules/internal/OwnedObject.chpl

```diff
@@ -194,23 +180,19 @@ module OwnedObject {
       Empty `obj` so that it manages `nil` and
       return the instance previously managed by this owned object.
 
-      If the argument is `nil` it returns `nil`.
-
-      .. note::
-         This is part of an new interface that will replace :proc:`owned.clear`.
-         However, `release` is not as widely used as `clear` yet, so there may
-         be some bugs we have not found yet. If you discover any bugs with
-         `release`, please report them to us and fall back on `clear`.
+      If the argument is `nil` it will halt
```

Comment:
We need to sort out the behavior when the argument is `nil`.

Reply:
Do we?  Is this any different than the current behavior with the old interface?  If not, it doesn't seem like a reason not to deprecate the old.